qubes/api/__init__.py

```diff
@@ -210,6 +210,17 @@ def enforce(predicate):
         if not predicate:
             raise PermissionDenied()
 
+    @classmethod
```

Comment:
You call it on an object instance, not class type, so it should be normal method.